.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: yarn workspace @shopify/polaris run validate-tokens
       - run: yarn workspace @shopify/polaris run type-check && yarn run build
-      - run: yarn workspace @shopify/polaris run lint
+      - run: yarn run lint
```

Comment:
Note: This `ci.yml` workflow was recently updated to only trigger on changes to `polaris-react`. Do we need to revert that to ensure all workspaces are being linted?

Reply:
I will note these improvements for later follow up in a separate issue. Thanks Ben!